### PR TITLE
ready for check then release

### PR DESCRIPTION
**Pull request by AlexBaranowski**

### CODE REVIEW

File: docs/release-notes/8.5.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+-   **Rust Toolset 1.54.0** - module `rust-toolset` stream `rhel`
+-   **Go Toolset 1.16.7** - module `go-toolset` stream `rhel`
+
+## Security
+
+Most of the security features are connected to rebased/updated versions
+of the packages. The most crucial security changes include:
+
+- rsyslog - now support OpenSSL
+- OpenSCAP framework - added new profiles and multiple changes and enhancement
```
Comment (tompods) [inline docs/release-notes/8.5.md:37]: enhancement**s**?

File: docs/release-notes/8.5.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+capital city of Albania.
+
+
+Below you can find the essential changes in EuroLinux 8.5 release for `x86_64`
+architecture. The `aarch64` GA release is slightly delayed. Release notes for
+`aarch64` will be provided in a separate document.
+
+## Dynamic programming languages, Web and Database servers
+
+Later versions of the following components are now available:
```
Comment (tompods) [inline docs/release-notes/8.5.md:16]: How about _New versions_?

File: docs/release-notes/8.5.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+## High Availability and Resilient Storage
+
+- cmirror was updated to version 2.03.12
+- corosync was updated to version 3.1.5
+- pacemaker was updated to version 2.1.0
+- resource-agents-paf package was added to HA an RS repositories
+
+## General updates and improvements
+
+Among the others, the following packages were updated and enhancement:
```
Comment (tompods) [inline docs/release-notes/8.5.md:50]: "and enhancement"?
Maybe _and enhanced_ if we're talking about the quality improvements these updates bring.

File: docs/release-notes/8.5.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+- scap-security-guide
+- skopeo
+- sos
+- subscription-manager
+- thunderbird
+
+
+## Batteries
+
+Batteries repository contains packages that are needed to rebuild distribution
```
Comment (tompods) [inline docs/release-notes/8.5.md:172]: _the distribution_ - we're talking about EuroLinux exclusively

File: docs/release-notes/8.5.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+includes more than 2100 packages for each buildroot.
+
+Multiple batteries were updated for this release there including:
+
+- Bootstraps for rust-toolset
+- Bootstraps for go-toolset
+
+## Gaia build system
+
+Gaia build system was updated in all interested parties environments. We also
```
Comment (tompods) [inline docs/release-notes/8.5.md:183]: _parties'_

File: docs/release-notes/8.5.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+changed the rebuild policy. Non of our customers were interested in using RHEL
```
Comment (tompods) [inline docs/release-notes/8.5.md:184]: _None_

File: docs/release-notes/8.5.md [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+
+
+## Other notable changes
+
+- EuroLinux 8.5 is the first version that can use `baseos-all`, `appstream-all` and
+  `powertools-all`,`'high-availability-all`, `resilient-storage-all`
+  repositories that contain **all** packages produced during build process.
+  However, these packages are not supported by upstream or EuroLinux. They
+  are intended for developers to build they own solutions.
+- EuroLinux reverted changes that made DockerHub default container image
```
Comment (tompods) [inline docs/release-notes/8.5.md:200]: _the changes_, _the default_?

**Review by tompods — CHANGES_REQUESTED** [review]
Nice one! Let's just fix some minor errors.

**Comment by AlexBaranowski** [comment]
@tompods fixed 😄

**Review by tompods — APPROVED** [review]

**Pull request merged**